can create contract.json

```diff
@@ -0,0 +1,113 @@
+    },
+    "client": {
+        "operator": {
+            "accountId": "0.0.47439",
+            "privateKey": "302e020100300506032b6570042204208f4014a3f7f7a6c7147070da98d88f9cea074c13ed0554783471825d801888cf"
+        },
+        "network": "testnet",
+        "mirrorNetwork": "testnet"
+    },
+    "test": [
+        {
+            "name": "fileCreateTransaction",
+            "request": {
+                "type": "FileCreateTransaction",
+                "data": {
+                    "keys": [ "$this.client.operatorKey" ],
```

Comment:
Values consistently take the form `"keyword(value)"`, which can be nested, `"keyword(keyword(value))"`

In this case, maybe `"keys": [ "operator_id()" ]`

Reply:
Keywords:
`literal(string value)` | `literal(0xFF)` | `literal(255)`
`constant(CONSTANT_NAME)`
`client_property(path.to.property)`

Response only:
`is_non_null()`
`is_null()`